src/encord_active/app/common/components/prediction_grid.py

```diff
@@ -34,7 +34,7 @@ def build_card_for_labels(
     data_dir: Path,
     label_color: Color = Color.RED,
 ):
-    class_colors = {int(k): idx["color"] for k, idx in get_state().predictions.all_classes.items()}
+    class_colors = {int(k): idx.get("color", Color.RED) for k, idx in get_state().predictions.all_classes.items()}
```

Comment:
is there another suitable name?

Reply:
what would you consider a suitable name?

Reply:
Something like ontology_object? idx especially in a loop is a bit confusing as it feels like it's an int

Reply:
ohh you meant `idx`, I thought you talked about `class_color` 🤦‍♂️